### PR TITLE
added czechcasting.com

### PR DESCRIPTION
**Pull request by bonobosexparty**

### CODE REVIEW

File: scrapers/czechcasting.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      Title: /html/head/title/text()
+      Details: //*[@id="girl-info"]/div/div/div[2]/p/text()
+      Tags:
+        Name: //*[@id="girl-info"]/div/div/ul[@class='tags']/li/a[@class='tag']/text()
+      Studio:
+        Name: $studio
+      Image:
+        selector: //video/@poster
+      Date:
+        selector: //p[@class='updated']/text()
```
Comment (bnkai) [inline scrapers/czechcasting.yml:21]: the date need to be in the format that stash expects so we need some extra postprocess
```suggestion
      Date:
        selector: //p[@class='updated']/text()
        postProcess:
          - parseDate: 01/02/2006
```

File: scrapers/czechcasting.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //*[@id="logo"]/a/img/@alt
+    scene:
+      Title: /html/head/title/text()
+      Details: //*[@id="girl-info"]/div/div/div[2]/p/text()
+      Tags:
+        Name: //*[@id="girl-info"]/div/div/ul[@class='tags']/li/a[@class='tag']/text()
```
Comment (bnkai) [inline scrapers/czechcasting.yml:15]:
```yaml
        Name: //ul[@class='tags']/li/a[@class='tag']/text()
```
is enough i think

File: scrapers/czechcasting.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    url:
+      - czechcasting.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //*[@id="logo"]/a/img/@alt
+    scene:
+      Title: /html/head/title/text()
+      Details: //*[@id="girl-info"]/div/div/div[2]/p/text()
```
Comment (bnkai) [inline scrapers/czechcasting.yml:13]:
```yaml
      Details: //div[@class="description show-desc"]
```
a bit cleaner

**Comment by bnkai** [comment]
Idk if it makes sense to add the performer names also in a page like this
```yaml
      Performers:
        Name:
          selector: //div[@id="girl-info"]//span[@class="name"]
          split: "/"
```
something like this seems to work ( for more than one performers in a scene  i only found one case that are seperated by `/` )

Again i am not sure if it makes sense to add this.

File: scrapers/czechcasting.yml (hunk 2 above)
Comment (bonobosexparty) [reply scrapers/czechcasting.yml:15]: Ah that makes sense. not sure why i was being overly specific

File: scrapers/czechcasting.yml (hunk 3 above)
Comment (bonobosexparty) [reply scrapers/czechcasting.yml]: added your suggestion

File: scrapers/czechcasting.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+name: czechav
```
Comment (bnkai) [inline scrapers/czechcasting.yml:1]: Just noticed that `czechav` name is used from the `czechav.yml` scraper.
Can you rename to `czechcasting` or something else?

**Pull request merged**